### DISCUSSION TITLE
TCP retransmissions

### DISCUSSION THREAD

**Discussion by knpshkn** [akvorado/akvorado#535] [Q&A] [answered]
Hello,

Is there a way to detect TCP retransmissions using sFlow? From what I understand, but I could be wrong, the main way to do this is to collect the TCP packet's sequence number/checksum and check if we already have packets with the same seqnum/checksum. I'm not sure if this would be appropriate, given that we usually have a sampling, but if there isn't, does it even make sense and is it possible to implement it in some way? Just curious about your thoughts on this

**Comment by vincentbernat** [comment, marked as answer]
Maybe it's possible, but this is highly improbable as most of the time, the sampling is done randomly. It would be possible to collect the sequence numbers, but then, you are on your own. Correlating packets seem way out of the scope of this project.
